src/main/scala/org/broadinstitute/dsde/workbench/leonardo/api/LeoRoutes.scala

```diff
@@ -51,46 +51,63 @@ abstract class LeoRoutes(val leonardoService: LeonardoService, val proxyService:
             }
           }
         } ~
```

Comment:
The route diffs look bigger than they are. I just added a top branch that checks for `v2`.

Reply:
`?w=1` helps with this